src/System.Device.Gpio/Interop/Windows/WinRT/Interop.WaitForCompletion.cs

```diff
@@ -8,7 +8,8 @@
 
 internal static partial class Interop
 {
-    public static TResult WaitForCompletion<TResult>(this IAsyncOperation<TResult> operation)
+    public static TResult? WaitForCompletion<TResult>(this IAsyncOperation<TResult> operation)
+        where TResult : class

```

Comment:
Just wondering, why did you have to add a constraint here?

Reply:
It's to deal with nullables being a very different feature between classes and structs.

Reply:
If you remove the constraint, you'll see the error. Would be interesting to see if you come to the same conclusion as me on the fix.

Reply:
I see yeah that's what I thought but I asked because I manually pulled down your PR branch and removed it and I was still able to build successfully, not sure why. Can you try removing it locally and rebuilding to see if you still get the warning?